packages/admin-test-utils/src/environment.ts

```diff
@@ -27,13 +27,35 @@ window.ResizeObserver = ResizeObserver;
  * If there's a prop type error then we want to throw an
  * error so that the test fails.
  *
- * NOTE: This can be removed once we move to a typescript
- * setup & we throw tests on type errors.
+ * In the CI we fail the entire process if anything is logged to the console.
+ * This is to stop pollution, you shouldn't need to log _anything_ to the console
+ * for tests.
  */
 
 const error = console.error;
+const warn = console.warn;
+const log = console.log;
+const info = console.info;
 window.console = {
   ...window.console,
+  info(...args: any[]) {
+    info(...args);
+    if (process.env.CI) {
```

Comment:
So a lot of the logs come from `act` or `axios` which aren't us specifically logging – this is either a warn or an error so this would be hard to have a lint warning for, tbf the TS Eslint config does ban consoles so we could omit `info` and `log`?

Reply:
I'm ok with this for now, it's a step up from what we had. But if anyone has any better ideas please give them :)

Reply:
We've removed info and log and now jest always throws :)